### ISSUE TITLE
CM6: Fix missing mergeview pickers

### ISSUE THREAD

**Issue by vidartf** [jupyter/nbdime#676]
After the CM6 upgrade, there is now missing picker handles in a few cases, especially for merge spacers. These should be re-added.

**Comment by vidartf** [comment]
Note, on #673, this was listed as:

> chosen differences or differences related to CM6
    - all padding spacers are directly displayed on the view, there is no need to click on a picker gutter markers to make a spacer fully expend with its real sizing.
    - there is no picker gutter marker for spacers. In the merge editor, only the text chunks are shown, there is no spacer.

I don't really understand this, as the pickers are meant to be used to pick which version of the code to use (local, remote or base).

**Comment by krassowski** [comment]
Somewhat related issue is that clicking on the gutter in a place which does not have a picker leads to an uncaught error:

```js
Uncaught TypeError: o is undefined
    onGutterClick mergeview.ts:1415
    mousedown mergeview.ts:1110
    ar index.js:9652
```

**Comment by HaudinFlorence** [comment]
> Note, on #673, this was listed as:
> 
> > chosen differences or differences related to CM6
> > - all padding spacers are directly displayed on the view, there is no need to click on a picker gutter markers to make a spacer fully expend with its real sizing.
> > - there is no picker gutter marker for spacers. In the merge editor, only the text chunks are shown, there is no spacer.
> 
> I don't really understand this, as the pickers are meant to be used to pick which version of the code to use (local, remote or base).

On the contrary, I don't understand why one needs to click on a picker to make the spacers appear. Maybe I missed something in the logics choosen for nbdime. I thought that the spacers are there to align the different views and are not related to a choice of what code will be picked.

**Comment by vidartf** [comment]
@HaudinFlorence My comment was mainly about the second point "there is no picker gutter marker for spacers. In the merge editor, only the text chunks are shown, there is no spacer". My understanding is that the pickers are needed also when there is added code vs an "empty spacer", so that you can pick the version of the code without the addition. For the first point, I'm not sure what it refers to, but that was at least never the intended behavior, and would sound like an indicator of a bug.

**Comment by krassowski** [comment]
> My understanding is that the pickers are needed also when there is added code vs an "empty spacer", so that you can pick the version of the code without the addition.

So that should be solved by #691 :)

**Comment by HaudinFlorence** [comment]
> @HaudinFlorence My comment was mainly about the second point "there is no picker gutter marker for spacers. In the merge editor, only the text chunks are shown, there is no spacer". My understanding is that the pickers are needed also when there is added code vs an "empty spacer", so that you can pick the version of the code without the addition. For the first point, I'm not sure what it refers to, but that was at least never the intended behavior, and would sound like an indicator of a bug.

@vidartf Ok sorry for the confusion and thanks for the explanation.
